config/queryOperation.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+require_once('config/connection.php');
+require_once ('config/session.php');
+
+class queryOperation
+{
+
+    public $conn;
+    public $connObj;
+    public $result;
+
+    /**
+     * Constructor function
+     *
+     * @access public
+     * @param  void
+     * @return void
+     */
+    function __construct()
+    {
+        // Creating the Connection class obj to make connection
+        $this->connObj = new connection();
+        $this->conn = $this->connObj->makeConnection();
+    }
+
+    /**
+     * Function for getting all details of employee
+     *
+     * @access public
+     * @param  int $id
+     * @return array
+     */
+    function getEmployeeDetail($id = '')
+    {
+        // To fetch the details for update, after log in
+        if (!empty($id))
+        {
+            $sqlQuery = "SELECT Employee.empId, Employee.title, Employee.firstName, Employee.middleName, 
+                Employee.lastName, Employee.email, Employee.phone, Employee.gender, Employee.dateOfBirth, 
+                Residence.street AS resStreet, Residence.city AS resCity , Residence.zip AS resZip, 
+                Residence.state AS resState, Office.street AS ofcStreet, Office.city AS ofcCity , Office.zip 
+                AS ofcZip, Office.state AS ofcState, Employee.maritalStatus AS marStatus, Employee.empStatus, 
+                Employee.image, Employee.employer, Employee.commId, Employee.note, Employee.password, Employee.note
+                FROM Employee 
+                JOIN Address AS Residence ON Employee.empId = Residence.empId 
+                AND Residence.addressType = 'residence'
+                JOIN Address AS Office ON Employee.empId = Office.empId 
+                AND Office.addressType = 'office'
+                HAVING EmpID = $id";
+        }
+        // To fetch the details to display
+        else
+        {
+            $sqlQuery = "SELECT Employee.empId AS EmpID,
+                CONCAT(Employee.title, ' ', Employee.firstName, 
+                ' ', Employee.middleName, ' ', Employee.lastName) AS Name,
+                Employee.email AS EmailID, 
+                Employee.phone AS Phone, Employee.gender AS Gender, Employee.dateOfBirth AS Dob, 
+                CONCAT(Residence.street, '<br />' , Residence.city , '<br />',
+                Residence.zip,'<br />', Residence.state ) AS Res,
+                CONCAT(Office.street, '<br />', Office.city , '<br />',  Office.zip, '<br />',
+                Office.state) AS Ofc,
+                Employee.maritalStatus AS marStatus, Employee.empStatus AS EmploymentStatus, 
+                Employee.employer AS Employer, Employee.commId AS Communication,
+                Employee.image AS Image, 
+                Employee.note AS Note
+                FROM Employee 
```

Comment:
FROM Employee 
JOIN Address AS Residence ON Employee.empId = Residence.empId AND Residence.addressType = 'residence'
JOIN Address AS Office ON Employee.empId = Office.empId AND Office.addressType = 'office'

Can put these in common var and use it.